url-shortener/src/main/java/com/bobocode/shortener/utils/UrlUtils.java

```diff
@@ -0,0 +1,13 @@
+package com.bobocode.shortener.utils;
+
+import lombok.experimental.UtilityClass;
+
+import java.net.URI;
+
+@UtilityClass
+public class UrlUtils {
```

Comment:
@vromantsev what's the point of this util class?

Reply:
Removed utility class.